doc/adrs/122_add_support_for_portal_representation.adoc

```diff
@@ -0,0 +1,109 @@
+
+=== Edit mode
+
+* In this mode, it's possible to change the position and the size of all the representations.
+New representations can also be added by dragging them from the explorer.
+* The representations can be reorganized by drag and drop action.
+All the representations can be resized via a handle placed in the bottom right corner of the representation.
+If the resize of a representation overlaps another one, this second representation is moved away.
+* A portal-type representation, has no horizontal or vertical boundaries.
+In case of overflow, there will be scrollbars for navigating.
+If a user doesn't want the page to be scrolled, it is possible by adjusting the layout.
+* Only in this mode, a representation can be deleted by using the _del_ key.
+This deletion does not affect the layout of the other representations.
+* When editing the _portal_, the representations are read-only.
+* Since a portal representation is itself a representation, it can be added to another _portal_.
+There is limitation to avoid a _portal_ including another _portal_ being added to this second _portal_.
+It prevents a _portal_ loop.
+
+=== Direct mode
+
+* In this mode, the representations can't be moved or resized.
+* Representations on a _portal_ have exactly the same behavior as if they were opened individually.
+A user can thus modify them.
+* If the representation provides one, the user has access to its palette.
+
+image:images/122/122_portal_representation_example.png[portal_representation_example]
+
+* If the defined size of a representation does not allow it to be displayed in its entirety, a scroll bar will be added in it.
+For diagram-type representations, it is possible to zoom inside.
+* Selecting an element from the explorer or from one of the _portal_ representations changes the selection in all representations.
+* If a representation is deleted but always referenced in a _portal_, its location is replaced by a blank slot with a message indication that the representation doesn't exist
+anymore.
+
+=== Representation header
+
+* Each representation on the _portal_ has a header on top of it, with the breadcrumb representing their path relative to the semantic element to which the portal representation is
+attached.
+
+image:images/122/122_representation_gantt_with_header.png[representation_with_header]
+
+=== Portal palette
+
+image:images/122/122_portal_palette.png[portal_palette]
+
+* The portal palette is always visible in direct and edit mode in the top left of the main zone.
+* The portal palette is composed of four buttons:
+
+. toggle to full screen
+. share button (opens a popup with the URL to access to the _portal_ representation)
+. toggle to edit mode
+. toggle to direct mode
+
+=== Representations panel
+
+* The right panel of the sirius page includes a representation view.
+* For _portal_, this view presents, in the form of a tree, the representations it contains.
+* No changes to a _portal_ content can be made from this view.
+
```

Comment:
The trash icon should be removed from the screen shot

Reply:
The last trash icons are for the main representation, it's the current behavior.
I only remove the one for the sub-representation